### DISCUSSION TITLE
New view; Combine Artwork and Compact

### DISCUSSION THREAD

**Discussion by JFizDaWiz** [TT-ReBORN/Georgia-ReBORN#53] [Ideas]
I would love to see a new view that combines both Artwork and Compact into one view.
Here's a quick mock-up on what I would love to see
![Untitled-1](https://user-images.githubusercontent.com/6844314/152451990-f20b8061-b449-4ba3-bf8f-fd50b1eb9621.png)

**Comment by TT-ReBORN** [comment]
Hi @JFizDaWiz,

thanks for your interest and idea, the only problem I see is that it's not really suitable for normal horizontal desktop usage because the height of the whole player would be really large. This would be only usable if people would switch their monitor to portrait mode...

Right now I have other important TODO's on my list that I want to finish, maybe if other users would be also interested I would consider to implement it.

-TT

**Reply by JFizDaWiz** [reply]
it doesn't need to be 1500px tall, it would be adjustable by making it shorter/longer by dragging a corner like it can be now

but your other point still stands yah, other important todo's

**Comment by silverwings15** [comment]
slightly off topic, but is this view possible in reBORN? 
![image](https://user-images.githubusercontent.com/10742312/152672923-44f1f230-af69-4fc5-aad2-0c2c47d5dfc7.png)

**Reply by TT-ReBORN** [reply]
It's possible if you implement it =)
This is the graphic browser from the Eole theme, the drop down track info menu is quite a cool feature. 
Maybe it will be implemented after the first public release...

**Reply by TT-ReBORN** [reply]
Btw, if you haven't checked it out, the library album art grid is already available if you go top menu:
Options > Library > Album art > Activate options or change design to album art.

Or for the quick switch, just right click in Library > Show album art. Right click again > Show tree.
If your in album art view, you can also right click and change width > "Change layout to full width" and change back to normal width.

-TT

**Reply by silverwings15** [reply]
yes i've been using that along with artist images as the art, great combo imo :D dropdown is definitely an awesome idea that would really synergize with the art grid view
